ros/src/i2c_com/scripts/i2c.py

```diff
@@ -1,17 +1,31 @@
 #! /usr/bin/python
 import rospy
+import time
+import smbus
+from BNO055 import BNO055
 from shared_msgs.msg import i2c_msg
+from shared_msgs.msg import imu_msg
+
+
 
 def message_received(msg):
   # This runs on a seperate thread from the pub
   pass
 
 if __name__ == "__main__":
   rospy.init_node('i2c_node')
-  pub = rospy.Publisher('i2c_rx', i2c_msg,
-      queue_size= 100)
-  sub = rospy.Subscriber('i2c_tx', i2c_msg,
-      message_received)
+  imu_pub = rospy.Publisher('imu_data', imu_msg,
+	qeue_size = 100)
+  imu_pub.rate(50) 
```

Comment:
Have rate as constant (Put 50 at top as constant), so we can more easily update it.